app/src/main/java/org/mozilla/fenix/AppRequestInterceptor.kt

```diff
@@ -116,5 +168,7 @@ class AppRequestInterceptor(private val context: Context) : RequestInterceptor {
     companion object {
         internal const val LOW_AND_MEDIUM_RISK_ERROR_PAGES = "low_and_medium_risk_error_pages.html"
         internal const val HIGH_RISK_ERROR_PAGES = "high_risk_error_pages.html"
+        internal const val AMO_BASE_URL = "https://addons.mozilla.org"
```

Comment:
Hey, quick question: would it be possible to configure this value with a pref?

Reply:
Hey, Definitely possible. Reason we didn't do it is that we wanted to limit installation to AMO. And also switching to `staging` and `dev` won't "just work", because the add-on IDs need to line up with the IDs from our collection. Is that what you had in mind?

We could introduce one we switch to the recommended API then? Another question would be if you'd want a build config and pref you can change at run-time. If it's the latter, we'd have to limit this to Nightly for now.